cli/handleValidate.js

```diff
@@ -1,4 +1,13 @@
-import { renderMissingConfig } from './templates';
+import path from 'path';
+import Ajv from 'ajv';
+import {
+  renderMissingConfig,
+  renderNativeError,
+  successMessage,
+  renderMissingConfigParameter,
+  renderWrongFormatForParam,
+} from './templates';
+import schema from './configuration.schema';
```

Comment:
I suggest we move `./cli/configuration.schema` to `./server/constants/config.schema.json`. You will find another file there, based on previous work I did. Please go ahead and overwrite it.